lib/config.rb

```diff
@@ -26,7 +27,7 @@ def url
     end
 
     def builds_dir
-      @builds_path ||= File.join($root_path, 'tmp', 'builds')
+      @builds_path ||= File.join($root_path, 'tmp', 'builds', 'runner-' + @runner.to_s)
```

Comment:
I just followed convention in the same file ...
See line below:
opts.on('-CWORKING_DIRECTORY', 'Specify the working directory for gitlab-ci-runner') do |v|

Anyway, I will fix single quotes